drizzlepac/runastrodriz.py

```diff
@@ -102,11 +103,11 @@
                  'resetbits': 4096}
 
 focus_pars = {"WFC3/IR": {'sigma': 2.0, 'good_bits': 512},
-              "WFC3/UVIS": {'sigma': 1.5, 'good_bits': ~14588},
-              "ACS/WFC": {'sigma': 1.5, 'good_bits': ~14588},
+              "WFC3/UVIS": {'sigma': 1.5, 'good_bits': 1360},
+              "ACS/WFC": {'sigma': 1.5, 'good_bits': 1360},
               "ACS/SBC": {'sigma': 2.0, 'good_bits': 0},
-              "ACS/HRC": {'sigma': 1.5, 'good_bits': ~14588},
-              "WFPC2/PC": {'sigma': 1.5, 'good_bits': ~14588}}
+              "ACS/HRC": {'sigma': 1.5, 'good_bits': 1360},
+              "WFPC2/PC": {'sigma': 1.5, 'good_bits': 1360}}
 sub_dirs = ['OrIg_files', 'pipeline-default']
```

Comment:
Add a comment which shows the individual components of 1360 and the associated meanings.  Also show the meaning of 4096 and 512.

Reply:
@mdlpstsci : Addressed in 591760f and 3c24be8.